testing/upload_validation.ts

```diff
@@ -92,6 +92,15 @@ import * as z from 'zod';
  */
 export const PACKS_VALID_COLUMN_FORMAT_MATCHER_REGEX = /^\/(.*)\/([a-z]+)?$/;
 
+export const Limits = {
+  BuildingBlockCountPerType: 100,
+  BuildingBlockName: 50,
+  BuildingBlockDescription: 250,
```

Comment:
this feels too small for description. maybe we should not limit the description field but can trim the description before indexing.

Reply:
It sounds like less but it's just the description of a particular building block, not the pack. Usually these are 1 sentence long but 250 allows for multiple sentences.

Reply:
250 could be the standard we enforce. if there's a situation when the pack maker needs more, they will report a bug. 

comparing to the other fields though, I just don't see this particular description length requirement as critical as others.